### ISSUE TITLE
PSO for fjsp

### ISSUE THREAD

**Issue by DrNTH** [ZhaoKe1024/IntelligentAlgorithmScheduler#3]
Thank you for sharing your project..may I ask if you solve the flexible Jobshop scheduling with the PSO algorithm??
you already used PSO for cloud scheduling but what about fjsp?

**Comment by ZhaoKe1024** [comment]
No, I didn't use PSO to solve the FJSP problem. I used genetic algorithm(GA) to solve it. In fact, particle swarm optimization(PSO) algorithm cannot solve combinatorial optimization problems, such as FJSP or cloud scheduling. Such problems usually use discretized particle swarm optimization (DPSO) to encode numerical values, such as binary encoding. It is similar to GA, where operations are also randomly modified on the right-hand side of the encodings. You can learn about DPSO and refer to GA to modify the code.

you can refer to my genetic algorithm in the ./JSSP_executor.py, and DPSO in the ./schedulers/DPSOTaskScheduling.py
